controllers/ControllerHome.php

```diff
@@ -19,9 +20,11 @@ public function __construct( $twig ) {
 	public function renderHome(){
 		$posts = new PostsManager();
 		$allPosts = $posts->listPosts();
-		if(!empty($_SESSION)){
+		$key = (new Session)->getKey('user');
+
+		if(!empty($key)){
 			$logged = true;
-			echo $this->twig->render( 'home.twig', ['allPosts' => $allPosts, 'logged'=> $logged, 'user'=> $_SESSION['user']['pseudo']]);
+			echo $this->twig->render( 'home.twig', ['allPosts' => $allPosts, 'logged'=> $logged, 'user'=> $key['pseudo']]);
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)